script/test_1.py

```diff
@@ -0,0 +1,13 @@
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+
+
+driver = webdriver.Chrome(executable_path="D:\Programowanie\chromedriver_win32\chromedriver.exe")
```

Comment:
It is better to put this executable in folder inside this project (e.g. `bin/`) and add this folder to .gitignore (So the file will not end up in the repo).